build-tools/packages/build-cli/package.json

```diff
@@ -123,7 +123,7 @@
 		"@fluid-private/readme-command": "workspace:~",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/eslint-config-fluid": "^3.2.0",
-		"@microsoft/api-extractor": "^7.38.3",
+		"@microsoft/api-extractor": "^7.39.1",
```

Comment:
Could be handled separately too but this package has redundant dev and non-dev deps on it.